.scalafmt.conf

```diff
@@ -0,0 +1,9 @@
+version = "3.5.9"
```

Comment:
Personally, I find inconsistent code style and especially super-long lines of code very hard to read. It is distracting and feels like an unnecessary cognitive effort which could be well spent somewhere else. Therefore, I'm still in favor of agreeing to a defined style and getting help from an auto-formatter like `scalafmt`. For Java Code we could use the integrated formatter of IDEA.

If we want to do this we need one big reformat of the code base and then stick to it.